eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/contig/ContigMappingTest.java

```diff
@@ -80,43 +74,37 @@ public void removePrefixOnlyAtTheBeginning() {
 
     @Test
     public void noSynonyms() {
-        assertNotNull(contigMapping.getContigSynonyms(CONTIG_WITHOUT_SYNONYM));
-        assertFalse(contigMapping.getContigSynonyms(CONTIG_WITHOUT_SYNONYM).isGenBankAndRefSeqIdentical());
-    }
-
-    @Test
-    public void noSynonymsOfAssignedMoleculeIfThereAreDuplicates() {
-        assertNull(contigMapping.getContigSynonyms(ContigMappingTest.ASSIGNED_MOLECULE_EXAMPLE));
-    }
-
-    @Test
-    public void getSynonymFromAssignedMoleculeIfThereAreDuplicatesOnlyInOtherKeys() {
-        assertNotNull(contigMapping.getContigSynonyms(ASSIGNED_MOLECULE_CONTIG_2).getAssignedMolecule());
+        ContigSynonyms contigSynonyms = contigMapping.getContigSynonyms(REFSEQ_WITHOUT_SYNONYM);
+        assertNotNull(contigSynonyms);
+        assertFalse(contigSynonyms.isIdenticalGenBankAndRefSeq());
+        assertEquals(SEQNAME_WITHOUT_SYNONYM, contigSynonyms.getSequenceName());
+        assertNull(contigSynonyms.getAssignedMolecule());
+        assertEquals(GENBANK_WITHOUT_SYNONYM, contigSynonyms.getGenBank());
+        assertEquals(REFSEQ_WITHOUT_SYNONYM, contigSynonyms.getRefSeq());
+        assertEquals(UCSC_WITHOUT_SYNONYM, contigSynonyms.getUcsc());
     }
 
     @Test
-    public void getAssignedMoleculeReturnsNullIfThereAreDuplicates() {
-        assertNull(contigMapping.getContigSynonyms(SEQNAME_CONTIG).getAssignedMolecule());
+    public void getSynonymsOfDuplicatedAssignedMolecule() {
+        ContigSynonyms contigSynonyms = contigMapping.getContigSynonyms(ASSIGNED_MOLECULE_CONTIG);
+        assertNotNull(contigSynonyms);
+        assertEquals(ASSEMBLED_MOLECULE_ROLE, contigSynonyms.getSequenceRole());
+        assertNotNull(contigSynonyms.getAssignedMolecule());
```

Comment:
Check the rest of synonyms.

Reply:
each of the rest of synonyms fetched from an "assigned molecule" have their own test below. this test is for testing the sequence role.